components/alert/alert.js

```diff
@@ -41,6 +42,7 @@ class Alert extends LocalizeStaticMixin(RtlMixin(LitElement)) {
 				left: 0;
 				margin: -1px;
 				min-width: 0.3rem;
+				position:absolute;
```

Comment:
Nitty nit: space

Reply:
Somehow I'm surprised we don't lint this.